src/utils/getResponse.ts

```diff
@@ -44,10 +44,19 @@ export const getResponse = async <
 
     const result = await handler.run(request)
 
-    if (result === null || !result.response || result.handler.shouldSkip) {
+    if (result === null || result.handler.shouldSkip) {
       return null
     }
```

Comment:
Hello, I don't know if it's really correct here.

I could have a handler with a `once` response that I think should be treated as handled, or not ?

Reply:
That's a valid concern. Note that `handler.shouldSkip` checks if the handler should be skipped. If you have a `res.once` runing _the first time_, `result.handler.shouldSkip` will be `false`. It's only later that it will be marked as skipped:

https://github.com/mswjs/msw/blob/12bdd274c2e283bdd16448813e58b510ec676ffc/src/utils/getResponse.ts#L51-L53

Reply:
yes right, in case I have a skipped request and I don't have any other handler rather than the one skipped I will have an error of unhandled?

Reply:
@marcosvega91 exactly. After `res.once` has executed once, it's ignored until you `.resetHandlers`, so it won't be taken into account when deciding if a request is handled or not.